deploy/chart/templates/0000_50_olm_10-operatorgroup.crd.yaml

```diff
@@ -6,9 +6,12 @@ spec:
   group: operators.coreos.com
   version: v1
   versions:
-  - name: v1
-    served: true
-    storage: true
+    - name: v1
```

Comment:
I assume YAML is forgiving with spacing, but it looks like the dash here isn't aligned as it was before. And the previous aligning is what the convention is.